index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <main id="main">
+      <h1 id="title" >Dr.Norman Borlaug</h1>
+      <div id="img-div">
+        <img id="image" src="project.jpg" style="max-width:100%; height:auto; align:center; display:block"/>
+        <div id ="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+        </div>
+        </div>
+        
+        
```

Comment:
kindly adjust your phrases, the comments should sound more like suggestions instead of orders. For example, _"I think it would be better to ..."._ 🤓👀